### PR TITLE
Raids and Groups - Organizer

### PR DESCRIPTION
**Pull request by kniteshift**
- [x] Create Mongoose Models for Raids
- [x] Create Mongoose Models for Groups
- [x] Create Class for Raids
- [x] Create Class for Groups
- [x] Allow a user to pull a list of raids based on the channel they are in
- [x] Allow a user to pull a list of raids based on the channel they are in and the Pokemon they want
- [ ] Allow user to setup a Group based on Raid ID and desired start time
- [ ] Allow users to pull a list of groups based on the channel they are in
- [ ] Allow users to pull a list of groups based on the channel they are in and the Pokemon they want
- [ ] Create an internal to expire and remove the expired Raids and Groups

In short, allows users to create lists of raids by channel, and groups by raid IDs.

### CODE REVIEW

**Comment by kniteshift** [comment]
Closing this because it doesn't follow the name space convention, sorry about that.
